test/test_battle.h

```diff
@@ -758,6 +766,8 @@ enum { TURN_CLOSED, TURN_OPEN, TURN_CLOSING };
 #define SKIP_TURN(battler) SkipTurn(__LINE__, battler)
 #define SEND_OUT(battler, partyIndex) SendOut(__LINE__, battler, partyIndex)
 
+#define WITH_RNG(tag, value) rng: ((struct TurnRNG) { tag, value })
```

Comment:
Because of the way the macros work out, the seemingly-unnecessary outer parentheses here are required to keep this whole thing being treated as a single token, which prevents `APPEND_TRUE` from inserting a `TRUE` between `tag` and `value`.

We could have instead chosen to have a 1-bit field that doesn't do anything in `TurnRNG` to absorb this value. This would mean we can write something more like `rng: { tag, value }` directly in `MOVE`, which might be nicer.